apps/spruce/src/pages/adminSettings/tabs/GeneralTab/ExternalCommunicationsTab/schemaFields.ts

```diff
@@ -109,7 +110,22 @@ export const slack = {
     },
     fieldsSet: {
       "ui:widget": widgets.ChipInputWidget,
+      "ui:description":
+        "If you specify a list of field names here, only those fields will be attached to the message. Note that this behavior does not apply if all fields is checked below.",
       "ui:fieldCss": fullWidthCss,
+      "ui:elementWrapperCSS": css`
```

Comment:
[nit] thoughts on removing this? 

with i feel like the chips are a little close 

<img width="894" height="283" alt="image" src="https://github.com/user-attachments/assets/f50305e9-060e-45bf-8eed-947e9f4bfdcd" />


without:


<img width="964" height="291" alt="image" src="https://github.com/user-attachments/assets/089497e0-8e9d-429a-ba52-ef154546c180" />